realm/realm-library/src/androidTest/java/io/realm/entities/pojo/AllTypesPojo.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.entities.pojo;
+
+import java.util.Date;
+
+import io.realm.RealmList;
+import io.realm.RealmModel;
+import io.realm.annotations.Index;
+import io.realm.annotations.PrimaryKey;
+import io.realm.annotations.RealmClass;
+import io.realm.entities.Dog;
+
+@RealmClass
+public class AllTypesPojo implements RealmModel {
```

Comment:
See above. I think using POJO is the wrong term. Perhaps use AllTypesRealmModel instead?